monkey/monkey_island/cc/setup/island_config_options.py

```diff
@@ -20,22 +19,39 @@
 
 
 class IslandConfigOptions:
-    def __init__(self, config_contents: dict = None):
-        if not config_contents:
-            config_contents = {}
-        self.data_dir = config_contents.get(_DATA_DIR, DEFAULT_DATA_DIR)
+    def __init__(self, config_contents: dict = {}):
+        self.data_dir = None
+        self.log_level = None
+        self.start_mongodb = None
+        self.crt_path = None
+        self.key_path = None
```

Comment:
Why is this needed when `self._initialize_defaults()` is called right after this?

Reply:
I tried this a few different ways. I didn't particularly like one version more than another. This was just the last way I tried it.

Members should be "defined" in the same place in every class so you don't need to go hunting around to figure out what members the class includes. I can just set the defaults in `_init_()`.